vnext/Microsoft.ReactNative/packages.lock.json

```diff
@@ -26,12 +26,9 @@
       },
       "Microsoft.UI.Xaml": {
         "type": "Direct",
-        "requested": "[2.8.0, )",
-        "resolved": "2.8.0",
-        "contentHash": "vxdHxTr63s5KVtNddMFpgvjBjUH50z7seq/5jLWmmSuf8poxg+sXrywkofUdE8ZstbpO9y3FL/IXXUcPYbeesA==",
-        "dependencies": {
-          "Microsoft.Web.WebView2": "1.0.1264.42"
-        }
+        "requested": "[2.7.0-prerelease.210913003, )",
+        "resolved": "2.7.0-prerelease.210913003",
```

Comment:
Why is this change generating impact to the lock file? It doesn't seem like it should be part of this PR.

Reply:
I had pulled from main into my branch - it may have been from that after running `yarn` as I was getting some errors locally after doing a rebase to remove some other commits that had unintentionally been included.